certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+              "title": "data",
+              "format": ""
+            }
+          }
+        ]
+      }
+    },
+    "/apiw/async/{resultId}/status": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Processing status of the specified result ID: PROCESSING:Rendering in progress, FAIL:Failure, SUCCESS:Success",
+                  "title": "status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "GetAsyncStatus",
+        "operationId": "GetAsyncStatus",
+        "description": "Get the processing status of the specified result ID.",
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the result ID returned by RenderAsync.",
+            "x-ms-summary": "Result ID to query processing status"
+          }
+        ]
+      }
+    },
+    "/apiw/async/{resultId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "title": "body",
+              "type": "string",
+              "format": "binary",
+              "description": "If the process is successful, the binary data of the generated form is returned. If the process fails, JSON with the details of the error is returned."
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "GetAsyncResult",
+        "description": "Get the processing result of the specified result ID.",
+        "operationId": "GetAsyncResult",
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the result ID returned by RenderAsync.",
+            "x-ms-summary": "Result ID of the processing result to be acquired."
+          }
+        ]
+      }
+    }
+  },
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://docurain.jp/"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://docurain.jp/privacypolicy/"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": ""
+    }
+  ],
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "API \u30ad\u30fc": {
+      "type": "apiKey",
+      "in": "header",
+      "name": "Authorization"
+    }
+  },
+  "security": [
+    {
+      "API \u30ad\u30fc": []
```

Comment:
@geckoR42 Please remove unicodes from the swagger

Reply:
@sriyen-msft Fixed "security"